x-pack/plugins/security_solution/public/common/components/navigation/solution_grouped_nav/solution_grouped_nav_panel.tsx

```diff
@@ -172,6 +173,7 @@ const SolutionNavPanelItems: React.FC<SolutionNavPanelItemsProps> = ({ items, on
           data-test-subj={`groupedNavPanelLink-${id}`}
           href={href}
           onClick={(ev) => {
+            track(METRIC_TYPE.CLICK, `${TELEMETRY_EVENT.GROUPED_NAVIGATION}${id}`);
```

Comment:
We already added the NAVIGATION track in the `item.onClick` function when we defined the items at:
https://github.com/elastic/kibana/blob/cbc52375e1cc7c91d8620c7d5d699aee0457bb6a/x-pack/plugins/security_solution/public/common/components/navigation/security_side_nav/security_side_nav.tsx#L77-L79

Or do we want to have another track for the GROUPED_NAVIGATION in this click? If that's the case I think it is better to add it in the `onClick` of the item definition outside.